frontend/src/components/PositionBar/PositionBar.js

```diff
@@ -4,11 +4,14 @@ import "./PositionBar.css"
 export default function PositionBar(props) {
     
     //TODO: implement distance formatter to keep zero padding
-
     return (
         <div className="position-bar-root">
-            <div className="position-bar-background"></div>
-            <div className="position-value">{props.distance}m</div>
+            <div className="position-bar-background">
+                <div className="cursor" style={{marginLeft: (props.telemetryData.crucial_data[0].value * 100) / (props.telemetryData.crucial_data[0].max) + '%'}}></div>
```

Comment:
Also, subtract the width of the cursor, because the curson is completely outside the bar after max value is reached.